source/_integrations/intellifire.markdown

```diff
@@ -65,3 +69,54 @@ The following is a description of the various diagnostic error sensors and what
  - **Offline** - Your appliance is currently offline.
  - **Pilot Flame** - Pilot Flame Error: Your appliance has been safely disabled. Please contact your dealer and report this issue.
  - **Soft Lock Out** - Sorry your appliance did not start. Try again by pressing Flame ON.
+
+### Has Errors Sensor
+
+Home assistant coding guidelines do not allow for a single sensor to represent multiple-different values - so although having a single sensor to indicate if there is an error state is a useful feature it must be added manually. This can be done by editing your `configuraiton.yaml` file and adding the following:
+
+```yaml
+binary_sensor:
+  - platform: template
+    sensors:
+      fireplace_has_errors:
+        friendly_name: "Fireplace Errors"
+        value_template: >
+          "{{ states('binary_sensor.fireplace_accessory_error')
+          or states('binary_sensor.fireplace_disabled_error')
+          or states('binary_sensor.fireplace_ecm_offline_error')
+          or states('binary_sensor.fireplace_fan_delay_error')
+          or states('binary_sensor.fireplace_fan_error')
+          or states('binary_sensor.fireplace_flame_error')
+          or states('binary_sensor.fireplace_lights_error')
+          or states('binary_sensor.fireplace_maintenance_error')
+          or states('binary_sensor.fireplace_offline_error')
+          or states('binary_sensor.fireplace_pilot_flame_error')
+          or states('binary_sensor.fireplace_soft_lock_out_error') }}"
+```          
```

Comment:
Creating template sensors should not be part of the integration guide. Additionally, groups are available for this.

Please remove this part.